src/babel/load-html-transformer.ts

```diff
@@ -0,0 +1,10 @@
+import { Transformer } from "@jest/transform";
```

Comment:
Consider making this single quotes for consistency in what I've seen so far in this PR

Reply:
Done